sqlmesh/core/console.py

```diff
@@ -441,10 +460,26 @@ def stop_creation_progress(self, success: bool = True) -> None:
         self.environment_naming_info = EnvironmentNamingInfo()
         self.default_catalog = None
 
+    def start_cleanup(self, ignore_ttl: bool) -> bool:
+        if ignore_ttl:
+            self._print(
+                "Are you sure you want to delete all orphaned snapshots regardless of their environment?"
+            )
+            if not self._confirm("This may affect other users. Proceed?"):
```

Comment:
This is not very API friendly

Reply:
True, and ironically it also makes it hard to test the Notebook magic because it requires user input.

However, I dont think this is something we should encourage the users to be running automatically anyway. It's more of an escape hatch to speed something up outside of the normal.

I could add a `--no-prompts` flag like some of the other operations, would that be better?